src/FlaUI.Core/AutomationElements/TextBox.cs

```diff
@@ -11,6 +12,25 @@ namespace FlaUI.Core.AutomationElements
     /// </summary>
     public class TextBox : AutomationElement
     {
+        internal class Win32Constants
+        {
+            public static IntPtr TRUE = new IntPtr(1);
+            public static IntPtr FALSE = new IntPtr(0);
+        }
+        
+        internal class WindowMessages
+        {
+            public static UInt32 WM_SETTEXT = 0x000C;
+            public static UInt32 WM_GETTEXT = 0x000D;
+            public static UInt32 WM_GETTEXTLENGTH = 0x000E;
+        }
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto)]
+        internal static extern IntPtr SendMessage(IntPtr hWnd, UInt32 Msg, IntPtr wParam, IntPtr lParam);
```

Comment:
There is a `WindowsAPI` namespace in FlaUI.Core, you should put that stuff in there.

Reply:
Done